aws-logs-metricfilter/src/main/java/software/amazon/logs/metricfilter/ResourceModelExtensions.java

```diff
@@ -0,0 +1,13 @@
+package software.amazon.logs.metricfilter;
+
+import org.json.JSONObject;
+
+public class ResourceModelExtensions {
+
+    public static JSONObject getPrimaryIdentifier(final ResourceModel model) {
+        final JSONObject identifier = new JSONObject();
+        identifier.append("LogGroupName", model.getLogGroupName());
+        identifier.append("FilterName", model.getFilterName());
+        return identifier;
```

Comment:
what is this for?

Reply:
This came with the PR I resurrected 😅  it's not being used. I'll remove it.